catalogData/redis32/3x-ha/k8s/service-proxy.json

```diff
@@ -0,0 +1,33 @@
+{
+  "apiVersion": "v1",
+  "kind": "Service",
+  "metadata": {
+    "labels": {
+      "org": "$org",
+      "space": "$space",
+      "catalog_service_id": "$catalog_service_id",
+      "catalog_plan_id": "$catalog_plan_id",
+      "service_id": "$service_id",
+      "idx_and_short_serviceid": "$idx_and_short_serviceid",
+      "managed_by": "TAP"
+    },
+    "name": "$idx_and_short_serviceid-proxy",
+    "annotations": {
+      "service.beta.kubernetes.io/aws-load-balancer-internal": "0.0.0.0/0"
```

Comment:
digging a little deeper, it looks like it's only used with a `type: LoadBalancer` so we can probably drop it from both. Looks like it was a remnant from when we had external IPs on these and was never removed. I haven't tried it without that setting as the proxies were never the issue once we got the actual proxy source working.

- https://github.com/18F/kubernetes-broker/commit/95d7aa172aeda557dcf2c93e0758c01c769404c1
- https://github.com/18F/kubernetes-broker/commit/a97159e4c5622b67e831534dfaf7180048d8d67d